index.js

```diff
@@ -357,6 +357,17 @@ class ScmRouter extends Scm {
     getDisplayName(config) {
         return this.scms[config.scmContext].getDisplayName();
     }
+
+    /**
+     * Get branch info of scmContext
+     * @method getBranchList
+     * @param  {Object}     config              Configuration
+     * @param  {String}     config.scmContext   Name of scm context
+     * @return {String}                         branch info of scmContext
+     */
+    getBranchList(config) {
```

Comment:
`_getBranchList`?

Reply:
You are right _getBranchList is correct. I will fix.